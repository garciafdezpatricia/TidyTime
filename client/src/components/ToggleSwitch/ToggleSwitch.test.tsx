import { render } from "@testing-library/react";
import '@testing-library/jest-dom';
import Toggle from '@/src/components/ToggleSwitch/ToggleSwitch';

describe('Toggle', () => {
    it('renders without crashing', () => {
      const { container } = render(<Toggle isChecked={false} onChange={() => {}}/>);
      expect(container).toBeInTheDocument();
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<Toggle isChecked={false} onChange={() => {}}/>);
        expect(asFragment()).toMatchSnapshot();
      });
  });