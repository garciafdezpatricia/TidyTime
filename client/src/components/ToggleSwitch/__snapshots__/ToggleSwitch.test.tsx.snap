// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggle matches snapshot 1`] = `
<DocumentFragment>
  <label
    class="switch"
  >
    <input
      type="checkbox"
    />
    <span
      class="slider round"
    />
  </label>
</DocumentFragment>
`;
